e2e/content-services/permissions-component.e2e.ts

```diff
@@ -36,8 +35,7 @@ describe('Permissions Component', function () {
     let permissionsPage = new PermissionsPage();
     let uploadActions = new UploadActions();
     let contentList = new ContentListPage();
-    let dataTablePage = new DataTablePage();
-    let acsUser, file;
+    let acsUser, acsUser2, file;
```

Comment:
would better use a more descriptive name instead of acsUser2? 
what is the characteristic of this second user?
why do we have another?
Maybe answering those question can drive a better name

Reply:
I have named the users as 'fileOwnerUser' and 'filePermissionUser'